qt_ui/windows/settings/QSettingsWindow.py

```diff
@@ -356,8 +358,9 @@ def applySettings(self):
             self.cheat_options.show_base_capture_cheat
         )
 
-        self.game.compute_unculled_zones()
+        self.game.compute_unculled_zones(GameUpdateEvents())
         GameUpdateSignal.get_instance().updateGame(self.game)
+        EventStream.put_nowait(GameUpdateEvents().update_unculled_zones())
```

Comment:
When changing culling options, these don't seem to get pushed unless we do it here. I would expect the `GameUpdateSignal` to take care of that, but that doesn't seem to be the case. I assume it's because `GameUpdateEvents` isn't static due to the dataclass decorator, which is probably why events seem to disappear. Therefore it could be interesting to drop the dataclass decorator so that it mimics a static class, though that could have unintended consequences I am not aware of yet.

Reply:
`GameUpdateSignal`'s existence is a bug, essentially. It's the source of a lot of bad UI performance because it's far too broad a signal. We don't want to increase reliance on it because we're actively trying to remove it (and the rest of Qt).